packages/kysely-sqld/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@libsql/kysely-sqld",
+
+  "type": "module",
+  "main": "lib-cjs/index.js",
+  "types": "lib-esm/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./lib-esm/index.d.ts",
+      "import": "./lib-esm/index.js",
+      "require": "./lib-cjs/index.js"
```

Comment:
I don't get why sometimes things come from the `lib-esm` dir, sometimes, from the `lib-cjs`?

Reply:
ES modules come from `lib-esm`, and CJS modules come from `lib-cjs`. TypeScript declarations are the same regardless of the target module system, so they are produced only in `lib-esm`.

Reply:
They are emitted here by the two invocations of `tsc` from `scripts.build-{cjs,esm}`, using `tsconfig.build-{cjs,esm}.json` as tsconfig.